test/controllers/director.residential.address.controller.unit.ts

```diff
@@ -29,7 +29,7 @@ const COMPANY_NUMBER = "12345678";
 const TRANSACTION_ID = "11223344";
 const SUBMISSION_ID = "55555555";
 const PAGE_HEADING = "Where does the director live?";
-const PUBLIC_REGISTER_INFORMATION = "What information we'll show on the public register";
+const PUBLIC_REGISTER_INFORMATION = "What information we'll show on the public online register";
```

Comment:
Since we are changing the text from multiple places, it would be nice to have one source of truth according to DRY. Maybe we could use a global environment variable. 

`nunjucksEnv.addGlobal('publicRegisterInformation', 'What information we'll show on the public online register')`